ash/src/vk/definitions.rs

```diff
@@ -51891,8 +51893,9 @@ unsafe impl<'a> TaggedStructure for GetLatencyMarkerInfoNV<'a> {
 }
 impl<'a> GetLatencyMarkerInfoNV<'a> {
     #[inline]
-    pub fn timings(mut self, timings: &'a mut LatencyTimingsFrameReportNV<'a>) -> Self {
-        self.p_timings = timings;
+    pub fn timings(mut self, timings: &'a mut [LatencyTimingsFrameReportNV<'a>]) -> Self {
+        self.timing_count = timings.len() as _;
+        self.p_timings = timings.as_mut_ptr();
```

Comment:
Note that the lifetime here makes the function useless, and we have to solve that in some way before merging this.

More context: https://github.com/ash-rs/ash/pull/815#discussion_r1392679017